NightscoutUploadKit/DeviceStatus/PredictedBG.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PredictedBG.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 8/8/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct PredictedBG {
+    let values: [Double]
+    let cob: [Double]?
+    let iob: [Double]?
+
+    public init(values: [Double], cob: [Double]? = nil, iob: [Double]? = nil) {
```

Comment:
NS uses the timestamp on the containing structure as the start date, and assumes a 5 minute interval thereafter. I could add a timestamp field to this, and we can later update NS to use it.  Do you think one base timestamp for all points, or separate base timestamps, one for each array (values, cob, iob)?  Seems like the former would suffice.